controller_manager/controller_manager/spawner.py

```diff
@@ -162,6 +162,12 @@ def main(args=None):
         raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), param_file)
 
     node = Node('spawner_' + controller_name)
+    spawner_namespace = node.get_namespace()
+    if spawner_namespace != '/' and not args.controller_manager.startswith('/'):
+        controller_manager_name = f"{spawner_namespace}/{args.controller_manager}"
+
+    controller_manager_name = make_absolute(controller_manager_name)
```

Comment:
we search for the controller manager in the same namespace as the spawner if the spawner has a namespace AND the controller manager name is not fully resolved.